charts/prometheus-node-exporter/values.yaml

```
@@ -317,6 +317,18 @@ hostRootFsMount:
   # https://kubernetes.io/docs/concepts/storage/volumes/#mount-propagation
   mountPropagation: HostToContainer
 
+# Mount the node's proc file system (/) at /host/proc in the container
+hostProcFsMount:
+  enabled: false
+  # Possible values are None, HostToContainer, and Bidirectional
+  mountPropagation: HostToContainer
```

As explained above, the toggle is not expected here as it does not enable/disable the mount in your code. 

```suggestion
hostProcFsMount:
  # Possible values are None, HostToContainer, and Bidirectional
  mountPropagation: ""
```

---

Change made